src/main/java/com/coyoapp/tinytask/web/TaskController.java

```diff
@@ -25,17 +29,18 @@
 class TaskController {
 
   private final TaskService taskService;
-
+  @Autowired
+  private TaskRepository taskRepository;
   @PostMapping
   public TaskResponse createTask(@RequestBody @Valid TaskRequest taskRequest) {
     log.debug("createTask(createTask={})", taskRequest);
     return taskService.createTask(taskRequest);
   }
 
   @GetMapping
-  public List<TaskResponse> getTasks() {
+  public List<Task> getTasks() {
     log.debug("getTasks()");
-    return taskService.getTasks();
+    return taskRepository.getTasks();
```

Comment:
Also just a hint as this is backend code: Do not access repositories directly from the controller, but use at least a device layer in between.